site2/website/static/swagger/2.4.0/swagger.json

```diff
@@ -5516,8 +5516,8 @@
     "/persistent/{tenant}/{namespace}/{topic}" : {
       "put" : {
         "tags" : [ "persistent topic" ],
-        "summary" : "Create a non-partitioned topic.",
-        "description" : "This is the only REST endpoint from which non-partitioned topics could be created.",
+        "summary" : "Create a partitioned topic.",
```

Comment:
I think the current doc is actually correct. this is the endpoint to create a non-partitioned topic

Reply:
Oh, yes, it's my fault.